crates/common/tedge_config/tests/test_tedge_config.rs

```diff
@@ -908,6 +908,7 @@ fn dummy_tedge_config_defaults() -> TEdgeConfigDefaults {
         default_http_bind_address: IpAddress(IpAddr::V4(Ipv4Addr::LOCALHOST)),
         default_c8y_smartrest_templates: TemplatesSet::default(),
         default_firmware_child_update_timeout: Seconds(3600),
+        default_service_type: String::from("service"),
     }
 }
```

Comment:
Update some existing tests like `test_parse_config_empty_file` to verify the defaults kicking in for this new setting.

Reply:
updated